### PR TITLE
Updating landing page map viz

### PR DESCRIPTION
**Pull request by ramyaragupathy**
landingpage

cluster

- [x] @ramyaragupathy to integrate tabs
- [x] @brendangatens to adjust country centroid position in the dataset directly
- [x] @brendangatens to style the centroids + tabs
- [x] @ramyaragupathy to integrate project density map
- [x] @brendangatens to style project density
- [x] @ramyaragupathy to integrate popup
- [x] @brendangatens to style project popup
- [x] Legend  update
- [x] Tab names update

cc @smit1678

### CODE REVIEW

**Comment by ramyaragupathy** [comment]
@brendangatens - just integrated the tab switch within the map view. Need your help to incorporate map div styling, particularly for project-view. Right now these two are stacked on one another and display styles are adjusted with tab buttons. Project view map does not stretch to the entire space, like the country-view. 

Update: switching to show/hide layers instead of multiple map divs

**Comment by ramyaragupathy** [comment]
- Added a blur property to the cluster layer to give a glow effect
- Individual points are differentiated with lesser opacity and a black outline
- Scroll zoom has been enabled
- mouse over on cluster layer changes cursor style

**Comment by ramyaragupathy** [comment]
@brendangatens @smit1678 - Like discussed, active project highlight is now available in country view. Approximately there are 60 active projects ( status change in <= 24 hrs) across 30 countries in the current file.

**Comment by ramyaragupathy** [comment]
Final changes:
landingpage

File: _layouts/project-index.html [hunk 1]
```diff
@@ -3,23 +3,57 @@
...
+        <div id = 'active-legend'>
+            <p><img src= 'images/dot-active.png' width="10" height="10"/><span  style="color:#D73F3F"> Active tasks</span></p>
+        </div>&nbsp;
+        <div id = 'member-legend'>
+            <p><img src= 'images/dot-blue.png' width="7" height="7"/> <span style="color:#929DB3"> Members</span></p>
         </div>
-        <div class="legend-item members">
-          <p>Members</p>
+        <!-- <div id = 'project-area-legend' class='center-hover-map-area hide'> -->
+            <div id = 'cluster-legend' class = "hide">
```
Comment (smit1678) [inline _layouts/project-index.html]: @ramyaragupathy Padding isn't completely even. If we use `<p>` tags instead we should get the same padding as the legend for the active countries.

**Comment by smit1678** [comment]
@ramyaragupathy @brendangatens what's the best way to get the plus centered? 

humanitarian openstreetmap team our work 2018-08-02 13-59-24

File: _layouts/project-index.html (hunk 1 above)
Comment (ramyaragupathy) [reply _layouts/project-index.html]: This was a good exercise for me in understanding padding details of different html elements 😸  Like you suggested, I've wrapped up all of the details into one `<p>` element

**Comment by ramyaragupathy** [comment]
@smit1678 - Thanks for reviewing! While you meant the offset in the plus symbol I thought it was the legend detail. I understand this is in the map data layer - so added an offset within the layout property. 

image

**Comment by ramyaragupathy** [comment]
Also checking if vertical arrangement in cluster page is better than the current horizontal arrangement. I think we should go with vertical arrangement of legends for project view because:
-  lengthier project titles highlights the padding difference between the hover and the legend. Placing the legend above one another makes the difference looks natural
- vertical arrangement gives more space for project details without making the page clumsier

image
image
image

**Comment by ramyaragupathy** [comment]
Per chat - replacing the css circles in the cluster legend with images 
landingpage

**Review by smit1678 — APPROVED** [review]

**Pull request merged**